app/__test__/page.test.tsx

```diff
@@ -0,0 +1,50 @@
+import userEvent from '@testing-library/user-event'
+import { render, screen, act, waitFor } from '@testing-library/react';
+import { Request, Response as NodeResponse } from 'node-fetch';
+import Home from '../page'
+import 'react-toastify/dist/ReactToastify.css';
+import { useRouter } from "next-router-mock";
+
+jest.mock("next/navigation", () => require("next-router-mock"));
+jest.spyOn(global, 'fetch').mockImplementation(() => 
+  Promise.resolve(
+    new Response(
+      JSON.stringify({
+        schemas: []  
+      }),
+      {
+        status: 200,
+        headers: {
+          'Content-Type': 'application/json'
+        }  
+      }
+    )
+  )
+)
+
+describe('Page', () => {
+    it('renders form fields', () => {
+        render(<Home />)
+        expect(screen.getByText('User(*):')).toBeInTheDocument()
+        expect(screen.getByText('Password(*):')).toBeInTheDocument()
+        expect(screen.getByText('Host(*):')).toBeInTheDocument()
+        expect(screen.getByText('Port(*):')).toBeInTheDocument()
+        expect(screen.getByText('Database(*):')).toBeInTheDocument()
+    })
+
+    it('displays error if form is invalid on submit', async () => {
+        render(<Home />)
+        expect(await screen.findByText('Fill All the required fields')).toBeInTheDocument()
```

Comment:
This testing scripts doesn't make sense. No action for submitting the form is described here.